docs/json-api.md

```diff
@@ -335,11 +335,30 @@ const styles = {
 Compiles to:
 
 ```css
-.button-jss-0-1 {
+.button-0-1 {
   color: '#0000B3'
 }
 ```
 
+## Alternative syntax using string templates
+
+You can use a template string literal or a regular string to declare the props and values.
+This notation is only supported for style declarations, nested rules, media queries, keyframes and others still have to be objects.
+
```

Comment:
Yes, thats why I say that we would need to export a function like `css` for e.g. from jss, while In wanted it to be a pure syntactic sugar implemented via plugin.